sympy/geometry/parabola.py

```diff
@@ -406,3 +407,21 @@ def vertex(self):
             vertex = Point(focus.args[0], focus.args[1] - self.p_parameter)
 
         return vertex
+from sympy import *
```

Comment:
No need to import here. All the imports must be at the top of the file. Avoid using `from sympy import *` here.